### DISCUSSION TITLE
Show album name above photo details

### DISCUSSION THREAD

**Discussion by chris-burrow-apps** [damongolding/immich-kiosk#99] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

To make it like for like with how photos are shown on Google Home Hub, please show the album name if the photo is in one.
Possibly in the bottom right with the other photo details or make it configurable so can be shown top/bottom/bottom right/etc.

**Comment by damongolding** [comment]
Added via `show_album_name` if one or more album IDs are specified.
